disperser/common/v2/blobstore/dynamo_metadata_store.go

```diff
@@ -480,6 +482,83 @@ func (s *BlobMetadataStore) GetBlobMetadataByRequestedAtBackward(
 	return result, lastProcessedCursor, nil
 }
 
+// GetBlobMetadataByAccountID returns blobs (as BlobMetadata) within time range (start, end)
+// (in ns, both exclusive), retrieved and ordered by RequestedAt timestamp in specified order, for
+// a given account.
+//
+// If specified order is ascending (`ascending` is true), retrieve data from the oldest (`start`)
+// to the newest (`end`); otherwise retrieve by the opposite direction.
+//
+// If limit > 0, returns at most that many blobs. If limit <= 0, returns all results
+// in the time range.
+func (s *BlobMetadataStore) GetBlobMetadataByAccountID(
```

Comment:
GetBlobMetadataByAccountID looks similar to GetDispersalRequestByDispersedAt, could we have some helper or better to keep the logic separate?

Reply:
It'll be nice to have a template or something, but just helping two functions seems not that high gain to take the complexity of template yet